module3/src/validators/group.ts

```diff
@@ -0,0 +1,14 @@
+import Joi from 'joi';
+import { availableGroupPermission } from '../models/group';
+
+export const groupSchema = Joi.object().keys({
+    name: Joi.string().required(),
+    permissions: Joi.array()
+        .items(
+            Joi.string().custom((value: string, helpers) => {
+                return availableGroupPermission.includes(value) ?
```

Comment:
you can use || - `availableGroupPermission.includes(value) || helpers.error(`There is no such permission as ${value}`)`

Reply:
Done